internal/providers/unix/components/process/controller.go

```diff
@@ -6,10 +6,11 @@ import (
 	"fmt"
 
 	"github.com/deref/exo/internal/util/jsonutil"
+	"github.com/goccy/go-yaml"
 )
 
 func (p *Process) InitResource() error {
-	if err := jsonutil.UnmarshalString(p.ComponentSpec, &p.Spec); err != nil {
+	if err := yaml.Unmarshal([]byte(p.ComponentSpec), &p.Spec); err != nil {
```

Comment:
1) why change this?
2) this will be a breaking change for any existing users

Reply:
1. To make it consistent with the docker-compose version.
2. I wouldn't think it would be a breaking change given that YAML is superset of JSON.

Reply:
1. not saying that it shouldn't be consistent, just wonder why it should be?
2. wow.... i had no idea that was true...

Reply:
ah, ok - i think i see your answer to #1 now: so you can do consistent handling with the load/save/etc without having to switch-on-type or soemthing like that. works for me.

Reply:
btw, I think there is a yamlutil.UnmarshalString too